.github/workflows/ci.yml

```diff
@@ -23,16 +23,16 @@ jobs:
         id: checktag
         run: |
           cd ${{ github.workspace }}
-          If($Env:GITHUB_REF -match "v([0-9]*)\.([0-9]*)\.([0-9]*)") {
+          If($Env:GITHUB_REF -match "(?<=v)([0-9]*)\.([0-9]*)\.([0-9]*)") {
```

Comment:
Git tags cannot start with numbers, so this change can only possibly handle impossible scenarios, therefore is redundant

Reply:
They actually can, thats unrelated to this change. The lookbehind excludes the `v` prefix from the regex match.

Reply:
Oh, they indeed can, it's only GitHub's release tag creator that has this limitation. TIL.